frontend/src/components/dashboard/ImgMediaCard.js

```diff
@@ -1,48 +1,46 @@
 import React from 'react';
 import {makeStyles} from '@material-ui/core/styles';
+import {Link} from 'react-router-dom';
 import Card from '@material-ui/core/Card';
 import CardActionArea from '@material-ui/core/CardActionArea';
 import CardActions from '@material-ui/core/CardActions';
 import CardContent from '@material-ui/core/CardContent';
 import CardMedia from '@material-ui/core/CardMedia';
 import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
-import PlaceholderImage from './images/placeholder-image.png';
+import PlaceholderImage from './images/paint.jpg';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) =>({
   root: {
     maxWidth: 345,
   },
-});
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.text.primary,
+  },
+}));
 
-export default function ImgMediaCard() {
+export default function ImgMediaCard({name, link}) {
```

Comment:
Now that you have props, it's good practice to have prop validation. You can see an example with gallery preview component.